Source/Scene/Cesium3DTileset.js

```diff
@@ -776,6 +777,17 @@ function Cesium3DTileset(options) {
    */
   this.backFaceCulling = defaultValue(options.backFaceCulling, true);
 
+  /**
+   * Whether to ignore the 3D model outline. When true, all outlines are not
+   * displayed. When false, outlines are displayed.
+   *
+   * @type {Boolean}
+   * @readonly
+   *
+   * @default false
+   */
+  this.showOutline = defaultValue(options.showOutline, true);
```

Comment:
Make sure the update the documentation - both the description and the `@default`

Reply:
Same comment for the `Model` documentation